src/crs_linter/rules/ctl_audit_log.py

```diff
@@ -0,0 +1,35 @@
+from crs_linter.lint_problem import LintProblem
+from crs_linter.rule import Rule
+
+
+class CtlAuditLog(Rule):
+    """Check there is no ctl:auditLogParts action in any rules."""
+
+    def __init__(self):
+        super().__init__()
+        self.success_message = "no 'ctl:auditLogParts' action found."
+        self.error_message = "Found 'ctl:auditLogParts' action"
+        self.error_title = "'ctl:auditLogParts' isn't allowed in CRS"
+        self.args = ("data",)
+
+    def check(self, data):
+        """check there is no ctl:auditLogParts action in any rules"""
+        for d in data:
+            if "actions" in d:
+                current_ruleid = 0
```

Comment:
Here you don't use the rule ID either so you should remove this line and below the block (or add rule ID to the message, for eg.).